src/main/java/com/spartronics4915/frc2019/subsystems/CargoChute.java

```diff
@@ -58,6 +58,7 @@ public static CargoChute getInstance()
     private TalonSRX mRampMotor = null;
     private Solenoid mRampSolenoid = null;
     private A21IRSensor mRampSensor = null;
+    
     private Timer mShootTimer = new Timer();
```

Comment:
Since we're no longer using this for just shooting, renaming (press f2) it to mCargoTimer would be good